x-pack/legacy/plugins/apm/public/components/app/ErrorGroupOverview/List/index.tsx

```diff
@@ -22,20 +20,8 @@ import {
   unit
 } from '../../../../style/variables';
 import { APMLink } from '../../../shared/Links/APMLink';
-import { fromQuery, toQuery } from '../../../shared/Links/url_helpers';
-import { history } from '../../../../utils/history';
-
-function paginateItems({
```

Comment:
@sqren this is where the pagination happens - that logic was also inside of ManagedTable.